### ISSUE TITLE
Trending page is too static

### ISSUE THREAD

**Issue by andzdroid** [Chocobozzz/PeerTube#508]
As far as I can tell, the Trending page is a list of videos with the most views of all time.

This makes the page very static and boring. It would be more interesting to see videos with the most views in the last 24 hours, or the last week, or month, etc.

**Comment by Chocobozzz** [comment]
Blocked by https://github.com/Chocobozzz/PeerTube/issues/262

**Comment by MayeulC** [comment]
Isn't that a duplicate of #330 ?
